js/components/Chip/Chip.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, TouchableOpacity, Text } from 'react-native';
+import { styles } from './styles';
+
+const Chip = ({ label, close }) => {
+  return (
+    <View style={{ flex: 1, flexDirection: 'row' }}>
```

Comment:
Please move inline CSS to the styles.js file

Reply:
done